tractor/_streaming.py

```diff
@@ -424,8 +424,17 @@ async def _maybe_raise_from_remote_msg(
                 f'Remote context error for {self.chan.uid}:{self.cid}:\n'
                 f'{msg["error"]["tb_str"]}'
             )
-            # await ctx._maybe_error_from_remote_msg(msg)
-            self._error = unpack_error(msg, self.chan)
+            error = unpack_error(msg, self.chan)
+            if (
```

Comment:
This is the crux of the fix and should cause some of our other context streaming tests to fail currently until adjusted to more correctly expect similar source errors.